src/Faker/Provider/Internet.php

```diff
@@ -331,4 +331,94 @@ public static function macAddress()
 
         return $mac;
     }
+
+    /**
+     *
+     */
+    const SMALLEST_HTML_LENGTH = 4; // <br>
+
+    /**
+     * @param string $tag
+     * @param string $content
+     * @param string $attributes as a string
+     *
+     * @return string
+     */
+    protected static function buildHtmlTag($tag, $content = '', $attributes = '')
+    {
+        $attributes = (0 === strlen($attributes)) ? "" : sprintf(" %s", $attributes);
+
+        return sprintf('<%s%s>%s</%s>', $tag, $attributes, $content, $tag);
+    }
+
+    /**
+     * @return array
+     */
+    protected static function getHtmlTags()
+    {
+
+        $tagList = array(
+           "div", "section", "article", "h1", "p", "i", "b", "span"
+        );
+
+        return $tagList;
+    }
+
+    /**
+     * Principe 0: No fixed chars length guaranteed, only a max size, as in Lorem::text()
+     * Principe 1: The more maxLength you want, the more html tags you get
+     * Principe 2: Available html tags list is fixed, for now
+     * Principe 3: Rendered HTML output is poorly semantic, for now.
+     *
+     * @param int $maxLength max string size expected
+     *
+     * @return string html output - html tags + Lorem text rendering
+     */
+    public static function html($maxLength = 100)
+    {
+        if (0 > ($maxLength - self::SMALLEST_HTML_LENGTH - 5)) { // 5 == Lorem::text() argument limit
```

Comment:
#noteToSelf: this repeated "5" value may become a const in the `Lorem` class, or at least a local variable